client/src/pages/MyPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react';
+import Header from 'src/components/common/Header';
+import Chat from 'src/components/menu/Chat';
+import SalesList from 'src/components/menu/SalesList';
+import WishList from 'src/components/menu/WishList';
+
+export default function MyPage() {
+  const [menu, setMenu] = useState('selesList');
```

Comment:
메뉴를 string으로 관리하지 않고, TypeScript의 Enum으로 사용하는 건 어떨까요?
```suggestion
  enum Menu {
    SalesList,
    Chat,
    WishList
  }

  const [menu, setMenu] = useState<Menu>(Menu.SalesList);
```

Enum으로 타입도 지정 되면서, 동시에 문자열을 상수화해주는 역할도 해서 좋은 것 같습니다!

Reply:
enum 배워갑니당. 수정해서 푸쉬했어요!!